Resources/translations/SonataAdminBundle.fr.xliff

```diff
@@ -86,6 +86,10 @@
                 <source>title_create</source>
                 <target>Créer</target>
             </trans-unit>
+            <trans-unit id="title_show">
+                <source>title_show</source>
+                <target>title_show</target>
```

Comment:
Currently the title is missing altogether. But the `link_action_show` reads as `Afficher`.